### ISSUE TITLE
Baselines shift a lot

### ISSUE THREAD

**Issue by davidfokkema** [HiSPARC/pysparc#57]
It seems that on PySPARC stations, the baseline is shifting a lot and is no longer at 200 ADC. Maybe it's temperature, maybe something else (dark current, light leaks?). Anyway, since the thresholds are fixed, the minimum required pulseheight to go over threshold is (much) reduced. This results in high event rates. Maybe look into the calibration procedure again? It is important to check if other (non-PySPARC) stations exhibit the same behaviour.

**Comment by 153957** [comment]
Is this a recent development or was this always a problem?

**Comment by davidfokkema** [comment]
I'm not sure. I think this has always been a problem for *some* stations. If that is because there is a small light leak or temperature dependence of the dark current of the PMTs, or a particular behaviour of the electronics box, I don't know.

**Comment by kaspervd** [comment]
The MIP-peak strongly shifts with temperature, we've seen changes from 110 mV to 220 mV and that's not rare. Because of this the single rates are very hard to interpret. I don't know about the baselines though..

In order to correctly interpret the pulse height distribution, all pulse heights need to be corrected for this change. I do this by measuring the MIP peak every 4 hours (trade-off between enough data and small enough time frame) and subsequently shifting this 4-hour distribution to a peak of 150 mV.

**Comment by davidfokkema** [comment]
Pulseheight (and thus the MIP peak) is measured from the baseline. So a shift in baseline would not necessarily mean a shift in MIP peak, and a shift in MIP peak cannot *only* be generated by a shift in baseline. However, it is clear that things shift. It could be the electronics, or it could be the PMT, or it could be both. Very annoying. We only have the baselines in the raw data, not the ESD, but I'll try to figure this out when I have some time.

**Comment by 153957** [comment]
Baselines are in the ESD data files.. just not in the CSV downloads.

Also @kaspervd if you are correcting/shifting pulseheights for MIP peaks, why dont you just plot the 'number of MIPs', since that is what you are actually doing.

**Comment by kaspervd** [comment]
Yes, I could also plot that on the x-axis. The important thing is to do the correction within a small time window (e.g. 4 hours) in which you assume the PMT temperature is roughly constant. So with a MIP-peak of 110 mV you interpret a 220 mV pulse height as 2 particles whereas it could really be just 1 particle but with a (temporary) over-enthousiastic PMT.

This PMT behaviour is reasonably well understood, if you see a day-night difference it is probably because of the PMT (or the electronic box is next to the window and in the Sun).
